src/components/App.js

```diff
@@ -33,13 +33,14 @@ const App = () => {
         <Route path="/login" component={LoginPage} />
         <Route path="/implicit/callback" component={LoginCallback} />
 
-        {/* any of the routes you need secured should be registered as SecureRoutes */}
-        <SecureRoute
+        {/* Removed the SecurerRoute component so that the HomePage loads first -JG */}
```

Comment:
What is the driver for this? Couldn't we introduce a new Landing page to be the root path?